star_tides/api/controllers/base_controller.py

```diff
@@ -0,0 +1,55 @@
+from abc import ABCMeta, abstractmethod
+from flask import request
+import base64
+
+class Controller(metaclass=ABCMeta):
```

Comment:
Unrelated: Please add a linter/formatter to the codebase please.

Reply:
Using pylint. Not sure what the best practices are in order to get them into the repo.